Rtpfec example

MathieuDuponchelle opened the pull request:
This depends on https://bugzilla.gnome.org/show_bug.cgi?id=792696

sdroege commented:
The 1.10 dependency should be conditional, like in the decodebin example. Many distros still have 1.8 unfortunately and you even update the requirement for all examples :)

MathieuDuponchelle commented:
> The 1.10 dependency should be conditional, like in the decodebin example. Many distros still have 1.8 unfortunately and you even update the requirement for all examples :)

Not sure why I needed it tbh, just removed it and things still build fine, updated the PR :)

sdroege commented on examples/src/bin/rtpfecclient.rs:
```diff
@@ -0,0 +1,290 @@
+    fecdec.set_property("storage", &internal_storage.to_value())?;
+    fecdec.set_property("pt", &100u32.to_value())?;
+
+    Ok (fecdec)
```
Please run this through rustfmt-nightly :)

sdroege commented on examples/src/bin/rtpfecclient.rs:
```diff
@@ -0,0 +1,290 @@
+
+    rtpbin.connect("new-storage", false, |values| {
+        let storage = values[1].get::<gst::Element>().expect("Invalid argument");
+        storage.set_property("size-time", &250000000u64.to_value()).unwrap();
```
This is looking for some underscores: `250_000_000`

sdroege commented on examples/src/bin/rtpfecclient.rs:
```diff
@@ -0,0 +1,290 @@
+            MessageView::Eos(..) => break,
+            MessageView::Error(err) => {
+                {
+                    match err.get_details() {
```
Hm this should be 1.10 API?

sdroege commented on examples/src/bin/rtpfecserver.rs:
```diff
@@ -0,0 +1,233 @@
+            MessageView::Error(err) => {
+                {
+                    match err.get_details() {
+                        Some(details) if details.get_name() == "error-details" => details
```
In both applications there is nothing that would ever have Some details. You don't post any error message with details anywhere

sdroege commented:
Doing the same example without FEC might also be nice for a as-simple-as-possible rtpbin example
